selinux/services/selinux.cil

```diff
@@ -0,0 +1,14 @@
+(allow kernel_t security_t (security (load_policy setbool setenforce setsecparam)))
+(allow init_t security_t (security (load_policy setbool setenforce setsecparam)))
```

Comment:
if we end up using `enforcing` kernel parameter, we're not going to use setenforce from init process, so it must be always denied for all so that enforce could never be removed in runtime.

Reply:
yes, `enforcing=1` will enforce even before init tries to setenforce. Thus I'll deny setenforce and load_policy for everyone, as we let kernel enforce it itself